src/include/update/rootpart.rb

```diff
@@ -63,12 +63,7 @@ def CanBeLinuxRootFS(partition_fs)
         return false
       end
 
-      # possible_root_fs contains list of supported FSs
-      # storage-ng
-      # FIXME: this kind of checks should be responsibility of the upcoming
-      # Y2Storage::FilesystemType (to be added as part of the
-      # "yast-storage-ng as a libstorage wrapper" change)
-      possible_root_fs = [:ext2, :ext3, :ext4, :btrfs, :reiser, :xfs]
+      possible_root_fs = Y2Storage::Filesystems::Type.root_filesystems().map { |x| x.to_sym }
       possible_root_fs.include?(partition_fs)
```

Comment:
I would actually prefer a method in Filesystems::Type like
Filesystems::Type::is_root_filesystem?. I don't expect a need to expose the actual list at all.